.tx/config

```diff
@@ -99,3 +99,9 @@ source_file = locale/af/accessibility-statement.json
 source_lang = af
 type = KEYVALUEJSON
 
+[suomifi-design-system.breadcrumb-json]
+file_filter = locale/<lang>/breadcrumb.json
+minimum_perc = 0
+source_file = locale/af/breadcrumb.json
+source_lang = af
+type = KEYVALUEJSON
```

Comment:
Should we keep empty lines at the end of all files, including config and json? At least there was an empty line before this PR.

Reply:
The empty lines themselves don't seem to be visible here, but the lack of them is indicated. The translation source file was the only one missing the empty line.